data/signersDefault.json

```diff
@@ -409,16 +415,22 @@
 			"sdid": "custhelp.com mailmw.custhelp.com",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL"
+		}, {
+			"domain": "datev.de",
+			"addr": "*",
+			"sdid": "datev.de",
+			"ruletype": "ALL",
+			"priority": "DEFAULT_RULE_ALL"
 		}, {
 			"domain": "decathlon.it",
 			"addr": "*@email.decathlon.it",
 			"sdid": "decathlon.it email.decathlon.it",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL_2"
 		}, {
-			"domain": "datev.de",
-			"addr": "*",
-			"sdid": "datev.de",
+			"domain": "degussa-bank.de",
+			"addr": "*@degussa-bank.de",
```

Comment:
Is it intentional that you exclude sub-domains from this rule (i.e. do not write just `*`)?

Same question for some of the other new rules.

Reply:
I've created the rules based on the mails I received. I'm not sure, that mails from all sub domains will be signed, also. But if you prefer it, I will just place stars...
It's up to you.

Reply:
Unless I know that that the domain is only signing parts of its mail, I normally just assume that all will be signed. But yes, this did cause sometimes problems in the past, there rules had to be adapted,

My assumption (mostly on my own usage) is that most people who have enabled the sign rules would rather have a false complain about a missing signature (and hopefully report it so it can get fixed), than a too relaxed rule that could result in a malicious mail to slip through.

But I'm fine with leaving as is too.